job.go

```diff
@@ -177,11 +177,18 @@ func (j *Job) ScheduledAtTime() string {
 
 // Weekday returns which day of the week the Job will run on and
 // will return an error if the Job is not scheduled weekly
-func (j *Job) Weekday() (time.Weekday, error) {
+func (j *Job) Weekday() ([]time.Weekday, error) {
```

Comment:
rather than changing the signature of this method, could we add `Weekdays()` and have that return the slice.

Then, have this method return the first weekday in the slice to maintain the old behavior of someone having only a single weekday scheduled?

Reply:
Sure, that can be done!

Reply:
I will also take a look into golangci

Reply:
Done!